packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -2747,6 +2747,26 @@ export abstract class AbstractMesh extends TransformNode implements IDisposable,
         return this;
     }
 
+    /**
+     * Optimize the indices order so that we keep the faces with similar indices together
+     * @returns the current mesh
+     */
+    public async optimizeIndicesAsync(): Promise<AbstractMesh> {

```

Comment:
Why return `this`? I personally find this pattern slightly worse because I have to actually read the doc comment to know whether it is returning a copy or the original mesh, whereas if it just returns void it is immediately obvious.

Reply:
It is to be more in line with other features that work the same way (like createNormals).